src/Makefile

```diff
@@ -955,6 +955,7 @@ TOOLS = xxd/xxd$(EXEEXT)
 #
 # Uncomment the next line to install Vim in your home directory.
 #prefix = $(HOME)
+prefix = $(HOME)/opt
```

Comment:
This probably isn't supposed to make it into the final version.

Reply:
Hm. Something is fishy: I don't seem able to mark this comment resolved. But it is. :)